frontend/src/Component/Share/Navbar/Sidebar.jsx

```
@@ -0,0 +1,66 @@
+
+                {/* Teacher Menu */}
+                {role === "teacher" && (
+                    <>
+                        <SidebarLink to="/teacher/dashboard" icon={HomeIcon} text="Dashboard" active={location.pathname.startsWith("/teacher/dashboard")} />
+                        <SidebarLink to="/teacher/manage-students" icon={UserGroupIcon} text="Manage Students" active={location.pathname.startsWith("/teacher/manage-students")} />
+                        <SidebarLink to="/teacher/grade-assignments" icon={ClipboardDocumentIcon} text="Grade Assignments" active={location.pathname.startsWith("/teacher/grade-assignments")} />
+                    </>
+                )}
+            </nav>
+
+            <button onClick={onLogout} className="flex items-center mt-auto text-red-400 hover:text-red-500">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)